sdks/python/tox.ini

```diff
@@ -296,7 +296,7 @@ commands =
   /bin/sh -c 'pytest -o junit_suite_name={envname} --junitxml=pytest_{envname}.xml -n 6 -m uses_pyarrow {posargs}; ret=$?; [ $ret = 5 ] && exit 0 || exit $ret'
 
 
-[testenv:py{38,39,310,311}-pandas-{14,15,20}]
+[testenv:py{38,39}-pandas-{14,15,20}]
```

Comment:
note: I removed 310/311 python versions from some configurations since we don't have any tests that exercise these configurations.  We might be upgrading to Python 3.9 soon, so left that for now.